Week01/Problem03/qxu_03.py

```diff
@@ -0,0 +1,12 @@
+def lpf(x):
+    i = 2
+    while i * i <= x:
+        if (x % i != 0):
+            i += 1
+        else:
+            x //= i
```

Comment:
Nice use of floor division...I had never seen that operator before, so I had to look it up.  Thanks!